src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -233,7 +208,37 @@ class HttpGoogleServicesDAO(
       }
     }
 
-    deleteGroups(bucketName) flatMap { _ => bucketDeleteFn() }
+    deleteGroups(bucketName) map { _ => monitorRef ! DeleteBucket(bucketName) }
```

Comment:
The BucketDeletionMonitor records in Orient that the bucket needs to be deleted, and calls back to the DAO to do the actual deletion